sdks/java/io/file-schema-transform/src/main/java/org/apache/beam/sdk/io/fileschematransform/JsonWriteSchemaTransformFormatProvider.java

```diff
@@ -51,22 +60,33 @@ public String identifier() {
    * {@link PCollection} file names written using {@link TextIO.Write}.
    */
   @Override
-  public PTransform<PCollection<Row>, PCollection<String>> buildTransform(
+  public PTransform<PCollection<Row>, PCollectionTuple> buildTransform(
```

Comment:
nit: please fix the comments at all places where we change the method args.

Reply:
Done